plugins/filter_kubernetes_sumologic.rb

```diff
@@ -40,9 +40,6 @@ def filter(tag, time, record)
 
       unless @source_category.nil?
         sumo_metadata[:category] = @source_category.dup
-        unless @source_category_prefix.nil?
-          sumo_metadata[:category].prepend(@source_category_prefix)
-        end
       end
```

Comment:
I am good with this change, however, we should cut a major release, as this changes compatibility and users may be dependent on how it works today (e.g. configuration in Sumo now requires updates.)